src/utils/clients.py

```
@@ -31,3 +48,271 @@
+    response_model: type[BaseModel] | None = None,
+    json_mode: bool = False,
+    max_tokens: int | None = None,
+    thinking_budget_tokens: int | None = None,
+    enable_retry: bool = True,
+    retry_attempts: int = 3,
+    stream: bool = False,
+    **extra_call_params: Any,
+) -> Callable[[Callable[P, Any]], Callable[P, Any]]: ...
+
+
+def honcho_llm_call(
```

Ruff (0.12.2)

`honcho_llm_call` is too complex (19 > 10)

(C901)